<!DOCTYPE html>
<html lang="en">

<head>

    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
    <meta name="description" content="">
    <meta name="author" content="">

    <title>DALS 2022</title>

    <!-- Bootstrap core CSS -->
    <link href="vendor/bootstrap/css/bootstrap.min.css" rel="stylesheet">

    <!-- Custom styles for this template -->
    <link href="css/scrolling-nav.css" rel="stylesheet">

</head>

<body id="page-top">

<!-- Navigation -->
<nav class="navbar navbar-expand-lg navbar-dark bg-dark fixed-top" id="mainNav">
    <div class="container">
        <a class="navbar-brand js-scroll-trigger" href="#page-top">DALS 2022</a>
        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive"
                aria-controls="navbarResponsive" aria-expanded="false" aria-label="Toggle navigation">
            <span class="navbar-toggler-icon"></span>
        </button>
        <div class="collapse navbar-collapse" id="navbarResponsive">
            <ul class="navbar-nav ml-auto">
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#about">About</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#topics">Topics of Interest</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#submission">Submission</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#special_issue">Special Issue</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#dates">Important Dates</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#Chairs">Chairs</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#committee">Committee</a>
                </li>
                <li class="nav-item">
                    <a class="nav-link js-scroll-trigger" href="#program">Program</a>
                </li>
                <!--                <li class="nav-item">-->
                <!--                    <a class="nav-link js-scroll-trigger" href="#collaborations">Collaborations</a>-->
                <!--                </li>-->
            </ul>
        </div>
    </div>
</nav>

<header class="text-black-50" style="background-image: url('imgs/scenario_bg.jpg'); min-height: 27rem;">
    <div class="container text-center">
        <h1>The 1<sup>st</sup> International Workshop <br> on Data Analysis in Life Science</h1>

        <!--        <img src="imgs/logo2022.png" alt="Logo CMLS" class="logo-cmls" width="320">-->

        <br><br>
        <h3>19-23 September, 2022 in Grenoble, France</h3>
        <h4 style="color:red;opacity: 0.7;">Online workshop</h4>
        <!--<h3 style="color:red">Held virtually</h3>-->

        <br>
        <h4>In conjunction with the
            European Conference on Machine Learning and Principles and Practice of Knowledge Discovery in Databases
            (ECML PKDD&nbsp;2022)</h4>
    </div>
</header>


<section id="about">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>About</h2>
                <p class="text-justify lead">

                    Data Scientists play a fundamental role in analyzing biological, genomics, and health care data, in
                    particular, to pave the ground to personalized medicine. Indeed, such data need to be properly
                    managed, integrated, and analyzed employing statistical inference tools as well as machine learning,
                    data mining, and deep learning methods. Indeed, an increasing number of data scientists are actively
                    working with bio-data, with many different goals, including patient stratification, personalized
                    medicine, drug, and therapy design. Furthermore, analyzing and mining the public large biological
                    and clinical databases (e.g.,
                    <a href="https://www.encodeproject.org/">ENCODE</a>,
                    <a href="https://sites.broadinstitute.org/ccle/">CCLE</a>,
                    <a href="https://portal.gdc.cancer.gov/">GDC</a>,
                    and
                    <a href="https://mimic.mit.edu/">MIMIC</a>)
                    have already proven to be paramount for
                    knowledge discovery. Thus, the Data Science community may benefit from learning and sharing the
                    various approaches that have been developed to deal with biological and clinical data.
                </p>
                <p class="text-justify lead">

                    The workshop will gather researchers with expertise in data management and analysis, machine
                    learning, and knowledge discovery applied to bioinformatics, healthcare, and life science problems.
                    This workshop aims to share current cutting-edge data science methodologies and their applications.
                    It will be an opportunity for researchers to meet and share their ideas on improving data-driven
                    personalized medicine, genetic data management, and health care system advancement. </p>
                </p>
            </div>
        </div>
    </div>
</section>

<section id="topics">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Topics of interest</h2>
                <p class="text-justify lead">The topics of interest include, but are not limited to:</p>
                <ul class="lead">
                    <li>Visualization and exploratory analysis methods for genomic and health care data</li>
                    <li>Data analysis of genomics and molecular biology data</li>
                    <li>Knowledge discovery from biological and clinical databases</li>
                    <li>Analysis of electronic health records (EHR)</li>
                    <li>Data analysis of heterogeneous health information systems and databases</li>
                    <li>Natural language processing methods applied to genomics and healthcare data</li>
                    <li>Tracking and visualization of the viral sequences</li>
                    <li>Machine learning methods applied to viral sequences</li>
                    <li>Deep learning methods for bioinformatics and biomedical images</li>
                    <li>Data pre-processing approaches for single-cell data</li>
                    <li>Single-cell data analysis</li>
                    <li>Data integration practices for precision medicine</li>
                    <li>Network-based approaches for genomic data analysis</li>
                </ul>
                <p class="text-justify lead">
                    We invite contributions from both industry and academia to share their research and experience in
                    using data science, machine learning and database knowledge practices with biological and clinical
                    data.
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for CMLS 2020:</p>
                <ul class="lead">
                    <li>Mireia Costa, Ana León, and Óscar Pastor. <i>The Importance of the Temporal Dimension in
                        Identifying Relevant Genomic Variants: a Case Study</i></li>
                    <li>Alberto García and Juan C. Casamayor. <i>Towards the Generation of a Species-Independent
                        Conceptual Schema of the Genome</i></li>
                    <li>Mohammed R. Elkobaisi, Heinrich C. Mayr, and Vladimir A. Shekhovtsov. <i>Conceptual Human
                        Emotion Modeling (HEM)</i></li>
                    <li>Pietro Crovari, Sara Pidò, and Franca Garzotto. <i>Towards an Ontology for Bioinformatics
                        Research Process</i></li>
                    <li>Giuseppe Agapito and Mario Cannataro. <i>Using BioPAX-Parser (BiP) to annotate lists of
                        biological entities with pathway data</i></li>
                    <li>Cristian Tristão and Antonio Basilio de Miranda, Edward Hermann Haeusler, Sergio Lifschitz. <i>Relational
                        Text-type for Biological Sequences</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>-->

<section id="submission">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Paper submission guidelines</h2>
                <p class="text-justify lead">
                    Papers must be written in English and formatted according to the
                    <a href="http://www.springer.com/gp/computer-science/lncs/conference-proceedings-guidelines">Springer
                        LNCS guidelines</a>.
                    <br>
                    Authors can submit their manuscript using the <a href="https://easychair.org/conferences/?conf=dals2022">EasyChair platforms.</a>.<br/>
                    Submissions may not exceed 12 pages in PDF format for full papers, respectively 6 pages for
                    short or demo papers, including figures and references.
                    Submitted papers must be original work that has not appeared in and is not under consideration for
                    another conference or journal. Work in progress is welcome, but preliminary results should be made
                    available as a proof of concept. Submissions only consisting of a proposal will be rejected.
                    <br>
                    At least one author should commit to present their work at the workshop.
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="cameraready">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <strike><h2>Camera-ready guidelines</h2></strike>
                <strike><p class="text-justify lead">
                    For the preparation of their manuscript camera-ready version, authors should consult <a
                        href="ftp://ftp.springernature.com/cs-proceeding/svproc/guidelines/Springer_Guidelines_for_Authors_of_Proceedings.pdf">Springer’s
                    authors’ guidelines</a>
                    and use their <a
                        href="https://www.springer.com/de/it-informatik/lncs/conference-proceedings-guidelines">proceedings
                    templates</a>, either for LaTeX or for Word,
                    for the preparation of their papers. The page limit for workshop papers is 10 pages, as for the
                    previously submitted version.
                    Springer encourages authors to include their ORCIDs in their papers.
                    <br><br>
                    In addition, the corresponding author of each paper, acting on behalf of all of the authors of that
                    paper,
                    must complete and sign a <a
                        href="https://drive.google.com/file/d/1FL5_jFrqayqbPRZ2nTuNKaY55_8L8xD8/view?usp=sharing">Consent-to-Publish
                    form</a>.
                    The corresponding author signing the copyright form
                    should match the corresponding author marked on the paper.
                    Once the files have been sent to Springer, changes relating to the authorship of the papers cannot
                    be made.
                </p></strike>
            </div>
        </div>
    </div>
</section>
-->


<section id="special_issue">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Post-conference publication</h2>
                <p class="text-justify lead">
                    <!--The papers accepted to the workshop's last edition CMLS 2022,
                    published within the Springer volume <a
                        href="https://link.springer.com/book/10.1007/978-3-030-65847-2">Advances in Conceptual
                    Modeling</a>,
                    have been invited to submit a revised and extended version
                    for a post-conference supplement in the journal
                    <a
                            href="https://bmcbioinformatics.biomedcentral.com/" target="_blank"><b>BMC
                        Bioinformatics</b></a>
                    journal (2-year IF 2.511, SJR 1.374).
                    We are working towards an agreement with the same journal also for this year.-->
                    Accepted papers will be published by Springer as joint proceedings of several ECML PKDD workshops.
                    
                    
                    We are working towards a preliminary agreement with <a
                        href="https://bmcbioinformatics.biomedcentral.com/" target="_blank"><b>BMC
                    Bioinformatics</b></a> journal (2-year Impact Factor: 3.169)
                    for a <b>post-conference supplement</b> related to Data modeling, Processing and Analysis for Life Sciences.
                    If the agreement is reached, all the papers accepted to our workshop will be invited
                    to submit a revised and extended version to the journal supplement.
                </p>
                <!--<p class="text-justify lead">For the <b>best paper</b> of the workshop (as assessed by the evaluations
                    of the members of the Program Committee) the <b>Article Processing Charges of the BMC Bioinformatics
                        supplement will be fully covered</b> by the organizers (GeCo at Politecnico di Milano & PROS at
                    Universitat Politècnica de València).-->
                </p>
            </div>
        </div>
    </div>
</section>


<section id="dates">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Important dates</h2>
                <p class="lead">
                The following deadlines are in AoE time zone (UTC – 12).
                <ul class="lead">
                    <li>Paper submission: 31 August 2022
                        <span style="color:red;opacity: 0.7;text-decoration: line-through;">20 June 2022</span>
                    </li>
                    <li>Notification: 10 September 2022
                        <span style="color:red;opacity: 0.7;text-decoration: line-through;">13 July 2022</span>
                    </li>
                    <li>ECML PKDD Conference dates: September 17-20, 2022</li>
                </ul>
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="registration">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Registration</h2>
                <p class="lead">
                    The registration for the workshop is open via the <a
                        href="https://er2020.big.tuwien.ac.at/registration/">ER website</a>.
                    At least one of the authors for each accepted paper must have an "Author Registration" (registration
                    fee of € 150,-.)
                    and attend the online conference for paper presentation.
                </p>
            </div>
        </div>
    </div>
</section>-->

<section id="Chairs">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Chairs</h2>
                <p class="lead">
                    <b>Arif Canakoglu</b>,
                    Dipartimento di Anestesia, Rianimazione ed Emergenza-Urgenza,
                    <br>Fondazione IRCCS Ca’ Granda Ospedale Maggiore Policlinico, Milan, Italy
                    <br>
                    <a href="mailto:arif.canakoglu@policlinico.mi.it">arif.canakoglu@policlinico.mi.it</a>
                </p>
                <p class="text-justify">
                    Arif Canakoglu currently works as a data scientist at Policlinico di Milano; and he works mainly on
                    the electronic health record of the intensive care ut's patients in the Lombardy region. He is
                    leading the research with the support of the medical group, analyzing the life quality of the
                    patients after the hospital discharge. Previously, he was involved in the “Data-driven Genomic
                    Computing” ERC Awarded project (2016-2021), where he contributed for developing integration of
                    heterogeneous genomic data and for developing computational methods for genomic applications. In
                    2016, He received his PhD on biomolecular knowledge data integration by using the modular schema
                    data warehouse. His research interests include data integration and data driven genomic computing,
                    big data analysis and processing on cloud computing, artificial intelligence applications. His main
                    areas of expertise are heterogeneous data integration and data driven models and machine learning
                    approaches in genomic, and big data processes, especially on cloud computing.
                <p><br><br>
                <p class="lead">
                    <b>Gaia Ceddia</b>, Life Sciences, Integrative Computational Network Biology
                    <br> Barcelona Supercomputing Center (BSC), Barcelona, Spain
                    <br>
                    <a href="mailto:gaia.ceddia@bsc.es">gaia.ceddia@bsc.es</a>
                </p>
                <p class="text-justify">
                    Gaia Ceddia is a Recognised Researcher at Barcelona Supercomputing Center. She is currently working
                    for the Integrative Computational Network Biology group directed by Prof. Nataša Pržulj. She
                    received his PhD cum laude in 2021, with a thesis titled “Computational Methods for Data-driven
                    Predictions and Understanding of Biological Interactions” at Politecnico di Milano. Her research
                    interests include the design of novel network science and machine learning algorithms to extract new
                    biomedical information and uncover molecular mechanisms of disease from biological data.
                </p><br><br>
                <p class="lead">
                    <b>Sara Pido</b>,
                    Laboratory for Information and Decision Systems (LIDS)
                    <br> Massachussets Institute of Technology, US
                    Dipartimento di Elettronica, Informazione e Bioingegneria (DEIB)
                    <br> Politecnico di Milano, Italy
                    <br>
                    <a href="mailto:sarapid@mit.edu">sarapid@mit.edu</a>
                </p>
                <p class="text-justify">
                    Sara Pido is a Ph.D. candidate at Politecnico di Milano in Data Analytics and Decision Sciences
                    supervised by the professor Stefano Ceri. She is currently a visiting student at Laboratory of
                    Information and Decision Systems, Massachusetts Institute of Technology, under the supervision of
                    Kalyan Veeramachaneni. She obtained her master degree at Politecnico di Milano under the supervision
                    of the professor Marco Masseroli with a thesis on the analysis of gene expression data through the
                    use of complex networks. She is currently working on drug repurposing techniques and on automatic
                    data science methods.
                </p><br><br>

                <p class="lead">
                    <b>Pietro Pinoli</b>,
                    Dipartimento di Elettronica, Informazione e Bioingegneria (DEIB)
                    <br> Politecnico di Milano, Italy
                    <br>
                    <a href="mailto:pietro.pinoli@polimi.it">pietro.pinoli@polimi.it</a>
                </p>
                <p class="text-justify">
                    Pietro Pinoli works as Researcher Fellow and lecturer at the Department of Electronics, Information
                    and Bioengineering at the Politecnico di Milano (Italy). He received his PhD cum laude in 2017, with
                    a thesis titled “Modeling and Querying Genomic Data” where he proposed and benchmarked data
                    structures and algorithms to manage, search and elaborate huge collections of genomic datasets, by
                    means of cloud and distributed technologies. He has been visiting PhD candidate at Harvard
                    University (Cambridge, MA, US). His research interests include bioinformatics and computational
                    biology, data bases and data management, big data technology and algorithms, machine learning and
                    natural language processing, and drug repurposing. He participated in the Italian PRIN GenData, ERC
                    GeCo and EIT VirusLab projects. In recent years he delivered talks at IBM Research in Almaden, Broad
                    Institute of Boston, University of Trento, University of Lausanne, IBM Research Zurich. He
                    co-organized tutorials sections at ER and EDBT, workshop at ICWE and he served as guest editor for
                    MDPI Biotech and BMC Supplements.
                </p><br><br>

            </div>
        </div>
    </div>
</section>

<section id="committee">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program Committee</h2>
                <p class="lead">
                    TBD
                    <!--<ul class="lead">
                        <li>Giuseppe Agapito, Magna Graecia University</li>
                        <li>Raffaele Calogero, University of Turin</li>
                        <li>Mario Cannataro, Magna Graecia University</li>
                        <li>Matteo Chiara, University of Milan</li>
                        <li>Johann Eder, University of Klagenfurt</li>
                        <li>Jose Luis Garrido, University of Granada</li>
                        <li>Giancarlo Guizzardi, Free University of Bozen-Bolzano</li>
                        <li>Carlos Íñiguez-Jarrín, National Polytechnic School</li>
                        <li>Sergio Lifschitz, Pontifical Catholic University of Rio de Janeiro
                        </li>
                        <li>Paolo Missier, Newcastle University</li>
                        <li>Ignacio Panach, University of Valencia</li>
                        <li>Barbara Pernici, Polytechnic University of Milan</li>
                        <li>Rosario Michael Piro, Polytechnic University of Milan</li>
                        <li>Sudha Ram, University of Arizona</li>
                        <li>Maria Rodriguez Martinez, IBM Zurich Research Laboratory</li>
                         <li>Pietro Pinoli, Politecnico di Milano</li>

                    </ul>-->
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="invited">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Invited Talk</h2>
                    <p class="text-justify">
                    We are glad to announce the CMLS 2020 Keynote Talk by <a
                        href="https://www.ncl.ac.uk/computing/people/profile/paolomissier.html#background">
                    Paolo Missier</a>: "<b>Optimising the re-execution of analytics pipelines in response to
                    changes in the data: current results, open problems, and opportunities</b>".
                    <br><br>
                    <a
                            href="https://www.ncl.ac.uk/computing/people/profile/paolomissier.html#background">
                        Paolo Missier</a> is Professor of Big Data Analytics
                    with the School of Computing at Newcastle University,
                    with 20 years experience in CS research, development, and research management.
                    <br>
                    The broad goal of his research is to understand the role of metadata, most notably data provenance,
                    in making sense of the underlying (big) data as well as improving and optimising the processes that
                    produce and extract added value from the data (i.e. through "big data" analytics)&mdash;he calls
                    this
                    metadata analytics.
                    <br>
                    He has been leading (as Principal Investigator) the ReComp project (2016-2019, EPSRC) focused on
                    preserving value from large-scale data analytics over time through selective re-computation
                    (<a href="http://recomp.org.uk/">http://recomp.org.uk/</a>)
                    where the challenge of collecting provenance metadata and extracting value from it through
                    analytics techniques is central to the research.
                    <br>

                    His talk will be focused on &mdash; but not limited to &mdash; the results of ReComp project and
                    explain
                    its application to Genomics and to the Simple Variant Interpretation (SVI) workflow.
                </p>
            </div>
        </div>
    </div>
</section>
-->

<section id="program">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Program</h2>
                <p class="lead">
                    TBD
                    <!--<h4>CMLS session is on Tuesday October 19th, 17:30-19:00 (CEST)</h4>
                    17:30-17:35
                    <br>Welcome and introduction to the workshop
                    <br>
                    <br>17:35-17:50
                    <br>Roman Lukyanenko, Veda Storey and Oscar Pastor Lopez:
                    <br>The Notion of “System” as a Core Conceptual Modeling Construct for Life Sciences.
                    <a target="_blank" class="pdf" href="presentations/1-TristaoEtAl.pdf">Slides</a>

                    <a target="_blank" class="video" href="https://youtu.be/Y74YXHj8-mE">Video</a>
                    <br>
                    </p>
                    <br>17.50-18.05
                    <br>Alberto García S., Mireia Costa Sánchez and Oscar Pastor Lopez:
                    <br>Characterization and Treatment of the Temporal Dimension of Genomic Variations: a Conceptual
                    Model-based approach.
                    <br>
                    <br>18.05-18.20
                    <br>Federico Comolli:
                    <br>Extension of the Genomic Conceptual Model to integrate Genome-Wide Association Studies.
                    <br>
                    <br>18.20-18.35
                    <br>Sara Balderas-Díaz, María José Rodríguez Fórtiz, Gabriel Guerrero-Contreras, Mercedes
                    Bellido-González and Jose Luis Garrido:
                    <br>Design of an Adaptable mHealth System Supporting a Psycho-Educational Program for Pregnant Women
                    with SGA Foetuses.
                    <br>
                    <br>18.35-19.00
                    <br>Oscar Pastor (<b>Keynote talk</b>):
                    <br>New directions for clinical and genomic problems in different diseases: A Conceptual Modeling-based
                    Approach-->
                </p>
            </div>
        </div>
    </div>
</section>

<!--<section id="accepted">
    <div class="container-fluid">
        <div class="row">
            <div class="col-lg-8 mx-auto">
                <h2>Accepted papers</h2>
                <p class="text-justify lead">
                    This is the list of accepted papers for CMLS 2021:</p>
                <ul class="lead">
                    <li>Roman Lukyanenko, Veda Storey and Oscar Pastor Lopez. <i>The Notion of “System” as a Core Conceptual Modeling Construct for Life Sciences</i></li>

                    <li>Alberto García S., Mireia Costa Sánchez and Oscar Pastor Lopez. <i>Characterization and Treatment of the Temporal Dimension of Genomic Variations: a Conceptual Model-based approach</i></li>

                    <li>Federico Comolli. <i>Extension of the Genomic Conceptual Model to integrate Genome-Wide Association Studies</i></li>

                    <li>Sara Balderas-Díaz, Rodríguez Fórtiz María José, Gabriel Guerrero-Contreras, Mercedes Bellido-González and Jose Luis Garrido. <i>Design of an Adaptable mHealth System Supporting a Psycho-Educational Program for Pregnant Women with SGA Foetuses</i></li>
                </ul>
            </div>
        </div>
    </div>
</section>-->

<!--<section id="collaborations">-->
<!--    <div class="container-fluid">-->
<!--        <div class="row">-->
<!--            <div class="col-lg-8 mx-auto">-->
<!--                <h2>Collaborations</h2>-->
<!--                This workshop is supported by the data-driven Genomic Computing group at Politecnico di Milano-->
<!--                and by the-->
<!--                <p class="lead">-->
<!--                    <a href="http://polimi.it/"><img src="imgs/polimi_white.jpg" alt="Logo CMLS" width="240"></a>-->
<!--                    <a href="http://www.bioinformatics.deib.polimi.it/geco/" target="_blank"><img-->
<!--                            src="imgs/geco_name.png"-->
<!--                            alt="Logo GeCo"-->
<!--                            width="170"-->
<!--                            class="mr-4"></a>-->
<!--                </p>-->
<!--            </div>-->
<!--        </div>-->
<!--    </div>-->
<!--</section>-->


<!-- Footer -->
<footer class="py-5 bg-dark">
    <div class="container">
        <div style="text-align: center;">
            <h2 class='text-light bg-dark'>Workshop on Data Analysis in Life Science</h2>
        </div>
    </div>
    <!-- /.container -->
</footer>

<!-- Bootstrap core JavaScript -->
<script src="vendor/jquery/jquery.min.js"></script>
<script src="vendor/bootstrap/js/bootstrap.bundle.min.js"></script>

<!-- Plugin JavaScript -->
<script src="vendor/jquery-easing/jquery.easing.min.js"></script>

<!-- Custom JavaScript for this theme -->
<script src="js/scrolling-nav.js"></script>

</body>

</html>
